### DISCUSSION TITLE
问个关于连接supabase数据库的问题

### DISCUSSION THREAD

**Discussion by HenryLogic** [Qexo/Qexo#466] [Q&A]
感谢作者的付出，其实用这个程序目前没什么问题。我倒是看到了文档中写的免费数据库supabase，就想创一个数据库连接typecho用。但typecho不知道为啥连不上supabase数据库提示如下，

对不起, 无法连接数据库, 请先检查数据库配置再继续进行安装: "SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known"

应该是找不到数据库。但我用作者您的程序在vercel上正确填写信息后可以连上，supabase那边也出现了内容。所以想问问解决方法。（其实感觉更应该到typecho下面问这个问题）

**Comment by am-abudu** [comment]
我并不了解，但我猜测是Typecho的程序设置问题。作者并未考虑到你会如此操作

**Reply by HenryLogic** [reply]
好的谢谢作者了，我去typecho下面问问。

**Comment by ethansongqc** [comment]
<img width="812" alt="微信图片_20250402132522" src="https://github.com/user-attachments/assets/5813c558-1b89-4365-a11c-a85d05cd89bb" />
<img width="526" alt="微信图片_20250402132534" src="https://github.com/user-attachments/assets/bf76e77b-c5ac-4fae-8495-ba3d5a084085" />
我想问下，这数据库要怎么连接，流程需要用到posatgres,里面的端口等参数都要等supabase连接上才显示

**Reply by am-abudu** [reply]
文档编写时没有这些东西，是直接进入supabase的网站查看。现在更建议使用Vercel自带的PostgreSQL数据库

**Reply by ethansongqc** [reply]
我解决了
